utils/runtime.go

```diff
@@ -0,0 +1,64 @@
+package utils
+
+import (
+	"context"
+
+	sharedpb "github.com/tron-us/go-btfs-common/protos/shared"
+	"github.com/tron-us/go-common/constant"
+	"github.com/tron-us/go-common/db/postgres"
+	"github.com/tron-us/go-common/db/redis"
+	"github.com/tron-us/go-common/env"
+	"github.com/tron-us/go-common/log"
+
+	"go.uber.org/zap"
+)
+
+func CheckRuntime(ctx context.Context, runtime *sharedpb.RuntimeInfoRequest) (*sharedpb.RuntimeInfoReport, error) {
+	// db runtime
+	report := new(sharedpb.RuntimeInfoReport)
+
+	report.Signature = runtime.Signature
```

Comment:
First RuntimeInfoRequest need to be update to move the signature outside of RuntimeInfoRequest, the signature is used to verify the request only.
The signature of return RuntimeInfoReport is signed for the RuntimeInfoReport, can not just "report.Signature = runtime.Signature", verify signature will fail.
Then where's signing using private key? If no private key here, the process who call this function need to sign after get the return RuntimeInfoReport.

Reply:
removed signature from runtime. process will deal with signing response.